composer.json

```diff
@@ -21,7 +21,7 @@
         "php": "^5.3.2 || ^7.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.5 || ^5.0.5"
+        "phpunit/phpunit": "^4.8.35 || ^5.7 || ^6.5"
```

Comment:
Is there any reason for the specific patch version (ie, why not 4.8.36, which is the latest). Wouldn't ^4.8 suffice?

Reply:
4.8.35 was the first PHPUnit 4.* version to support namespaces

Reply:
Ok, thanks for clarifying